packages/trader/src/Assets/Trading/Categories/turboslong-trade-description.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+
+export const TurbosLongTradeDescription = () => {
+    const content = [
+        {
+            type: 'paragraph',
+            text: localize(
+                'Predict the market direction, size and also how confident you are that the spot price will not touch the barrier.'
+            ),
+        },
+        {
+            type: 'paragraph',
+            text: localize(
+                'If you select “Long”, you will earn a payout if the spot price never touches the barrier, that is the spot price is always above the barrier. Your payout will grow proportionally according to the distance between the market price and the barrier, with the condition that the spot didn’t cross the barrier at any time during the trade. If the spot price touches or breaches the barrier during the trade, then there won’t be a payout. Select a “Long” contract if you think the market will grow strongly without falling.'
+            ),
+        },
```

Comment:
if only 1 paragraph differs:
```suggestion
export const TurbosTradeDescription = (is_turbos_long) => {
    const second_paragraph = is_turbos_long
            ? localize(
                'If you select “Long”, you will earn a payout if the spot price never touches the barrier, that is the spot price is always above the barrier. Your payout will grow proportionally according to the distance between the market price and the barrier, with the condition that the spot didn’t cross the barrier at any time during the trade. If the spot price touches or breaches the barrier during the trade, then there won’t be a payout. Select a “Long” contract if you think the market will grow strongly without falling.'
            )
            : localize(
                'If you select “Short”, you will earn a payout if the spot price never touches the barrier, that is the spot price is always below the barrier. Your payout will grow proportionally according to the distance between the market price and the barrier, with the condition that the spot didn’t cross the barrier at any time during the trade. If the spot price touches or breaches the barrier during the trade, then there won’t be a payout. Select a “Short” contract if you think the market will decay strongly without rising.'
            );
    const content = [
        {
            type: 'paragraph',
            text: localize(
                'Predict the market direction, size and also how confident you are that the spot price will not touch the barrier.'
            ),
        },
        {
            type: 'paragraph',
            text: second_paragraph,
        },
```

Reply:
If the difference between final text will be minor, I'll reuse that component.